Source/Orts.Simulation/Simulation/Activity.cs

```diff
@@ -1502,17 +1518,23 @@ override public Boolean Triggered(Activity activity)
         {
             var triggered = false;
             var e = this.ParsedObject as Orts.Formats.Msts.EventCategoryLocation;
+            Train train = Simulator.PlayerLocomotive.Train;
```

Comment:
```suggestion
            var train = Simulator.PlayerLocomotive.Train;
```

Reply:
OK. Should I click on "Commit suggestion"? Do I have then to make the same change in my local and remote copy to keep everything aligned?

Reply:
I believe the "Commit suggestion" will modify the branch in GitHub with my suggested change, so afterwards you will need to do `git pull` to get the change locally.